carml-engine/src/test/resources/RmlMapper/nullValues/nullValueReference.rml.ttl

```diff
@@ -0,0 +1,29 @@
+@prefix rr: <http://www.w3.org/ns/r2rml#>.
+@prefix rml: <http://semweb.mmlab.be/ns/rml#>.
+@prefix ex: <http://example.com/>.
+@prefix ql: <http://semweb.mmlab.be/ns/ql#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<#TermTypeMappingLiteral> a rr:TriplesMap ;
+  rml:logicalSource [
+    rml:source "nullValues.json" ;
+    rml:referenceFormulation ql:JSONPath ;
+    rml:iterator "$.Child[*]" ;
+  ] ;
+
+  rr:subjectMap [
+    rr:template "http://example.com/Child/{last}" ;
+    rr:class ex:Child ;
+  ] ;
+
+  rr:predicateObjectMap [
+    rr:predicateMap [
+      rr:constant ex:hasBirthday ;
+    ] ;
+    rr:objectMap [
+      rml:reference "birthday" ;
+      rr:datatype xsd:date ;
+      rr:termType rr:Literal ;
+    ] ;
```

Comment:
Would it make sense to add `first` in a predicate object map here? Since it is null, the expected result should be the same.

Reply:
I don't think I follow

Reply:
like in the other test mapping `nullValueMultiValuedTemplate.rml.ttl`:

```ttl
 rr:predicateObjectMap [
    rr:predicate ex:knows ;
    rr:objectMap [
      rr:template "http://example.com/Child/{$.Child.*.first}" ;
    ] ;
  ] ;
```

Reply:
Ok, great!